cmd/core/pfcp_session_handlers_test.go

```diff
@@ -293,3 +297,154 @@ func TestSdfFilterStoreInvalid(t *testing.T) {
 		t.Errorf("Bad SDF shouldn't be stored")
 	}
 }
+
+func TestFTUPInAssociationSetupResponse(t *testing.T) {
+
+	config.Conf = config.UpfConfig{
+		IPPool:      "10.61.0.0/16",
+		FTEIDPool:   65536,
+		FeatureUEIP: false,
+		FeatureFTUP: true,
+	}
+
+	pfcpConn, smfIP := SdfFilterStorePreSetup(t)
```

Comment:
Strange name for this test. Rename to be more generic or make a copy with appropriate name